src/overlays/actors/ovl_En_Ot/z_en_ot.h

```diff
@@ -7,10 +7,61 @@ struct EnOt;
 
 typedef void (*EnOtActionFunc)(struct EnOt*, GlobalContext*);
 
+#define ENOT_GET_7F(thisx) ((thisx)->params & 0x7F)
+#define ENOT_GET_3F80(thisx) (((thisx)->params >> 7) & 0x7F)
+#define ENOT_GET_3FFF(thisx) ((thisx)->params & 0x3FFF)
+#define ENOT_GET_C000(thisx) (((thisx)->params >> 0xE) & 3)
+
+typedef struct {
+    /* 0x00 */ u8 unk_00;
+    /* 0x04 */ f32 unk_04;
+    /* 0x08 */ char unk_08[0x4];
+    /* 0x0C */ Vec3f unk_0C;
+    /* 0x18 */ char unk_18[0x18];
+    /* 0x30 */ f32 unk_30;
+    /* 0x34 */ f32 unk_34;
+    /* 0x38 */ f32 unk_38;
+    /* 0x3C */ f32 unk_3C;
+    /* 0x40 */ f32 unk_40;
+    /* 0x44 */ char unk_44[0x8];
```

Comment:
```suggestion
    /* 0x08 */ UNK_TYPE1 unk_08[0x4];
    /* 0x0C */ Vec3f unk_0C;
    /* 0x18 */ UNK_TYPE1 unk_18[0x18];
    /* 0x30 */ f32 unk_30;
    /* 0x34 */ f32 unk_34;
    /* 0x38 */ f32 unk_38;
    /* 0x3C */ f32 unk_3C;
    /* 0x40 */ f32 unk_40;
    /* 0x44 */ UNK_TYPE1 unk_44[0x8];
```

Reply:
We don't do this with any other struct?

Reply:
I thought this was pretty standard when it comes to MM, once part of a struct is known to be unused, we replace it with `UNK_TYPE1`